exonum/src/helpers/fabric/details.rs

```diff
@@ -57,9 +57,12 @@ impl Run {
         Box::new(RocksDB::open(Path::new(&path), options).expect("Can't load database file"))
     }
 
-    fn node_config(ctx: &Context) -> NodeConfig {
-        let path = ctx.arg::<String>(NODE_CONFIG_PATH)
-            .unwrap_or_else(|_| panic!("{} not found.", NODE_CONFIG_PATH));
+    fn node_config_path(ctx: &Context) -> String {
+        ctx.arg::<String>(NODE_CONFIG_PATH)
+            .unwrap_or_else(|_| panic!("{} not found.", NODE_CONFIG_PATH))
+    }
+
+    fn node_config(path: String) -> NodeConfig {
```

Comment:
`P: AsRef<Path>` is better

Reply:
We discussed this topic with @vitvakatu and decided to left `String` here, so interface of `Node::new` won't become uglier.